webapp/src/components/jira_issue_selector/jira_issue_selector.jsx

```diff
@@ -83,6 +83,24 @@ export default class JiraIssueSelector extends Component {
             );
         }
 
+        const serverError = this.state.error;
+        let errComponent;
+        if (this.state.error) {
+            errComponent = (
+
+                <React.Fragment>
```

Comment:
I'm not sure if you need this to be a fragment -- I think you can just return a p

Reply:
Verified this is not needed.  Removing